src/ui/components/shared/SharingModal/SharingModal.tsx

```diff
@@ -87,19 +88,44 @@ function CollaboratorRequests({ recording }: { recording: Recording }) {
   );
 }
 
-function CollaboratorsSection({ recording }: { recording: Recording }) {
+function CollaboratorsSection({
+  recording,
+  showPrivacy,
+  setShowPrivacy,
+}: {
+  recording: Recording;
+  showPrivacy: boolean;
+  setShowPrivacy: React.Dispatch<React.SetStateAction<boolean>>;
+}) {
   const { hasNoRole, loading } = useHasNoRole();
 
   if (hasNoRole || loading) {
     return null;
   }
 
   return (
-    <section className="space-y-4 bg-themeBase-100 p-8">
+    <section className="space-y-4 bg-modalBgcolor p-4">
       <div className="flex w-full flex-col justify-between space-y-3">
         <div className="w-full space-y-4">
-          <div className="space-y-1.5">
-            <div className="font-bold">Add People</div>
+          <div>
+            <div className="mb-2 font-bold">Team</div>
+
+            <div className="rounded-md border border border-transparent bg-themeTextFieldBgcolor p-2 hover:bg-themeTextFieldBgcolorHover">
+              <PrivacyDropdown {...{ recording }} />
```

Comment:
Unnecessary spread?
```suggestion
              <PrivacyDropdown recording={recording} />
```

Reply:
Good call, not actually sure because this is older code. But I trust you! Committed.

Reply:
The expression `...{ recording }` is doing two things:
1. Creating a new object with the property "recording" that points to the value of whatever is in that variable.
2. _Spreading it_ as props passed to `PrivacyDropdown` (one prop per attribute– so a `recording` prop).

So this code:
```jsx
<PrivacyDropdown {...{ recording }} />
```

And this code:
```jsx
<PrivacyDropdown recording={recording} />
```

Will _both_ get compiled to something like this code:
```js
createElement(PrivacyDropdown, { recording });
```

The second one is just easier to read.